.github/workflows/codeql.yml

```diff
@@ -15,6 +15,8 @@ on:
   push:
   pull_request:
     branches: [ master ]
+    paths-ignore:
+      - '**/*_min.js'
```

Comment:
This will hide any file ending of "_min.js" which must not be necessary a minimized JavaScript version of a bigger JavaScript file. Other minimized JavaScript files which have a other name schema for minimization will be checked still. If I would to ignore a file I would name it directly and not use a global big match pattern. But this is only my opinion and can be ignored if they did not right.

Reply:
I think jquery-2.1.1.min.js should be removed because it is a ten years old security risk. Maybe Kitodo can use the JQuery package which is part of Debian / Ubuntu if this is required.